controllers/gce/controller/controller.go

```diff
@@ -279,7 +280,13 @@ func (lbc *LoadBalancerController) sync(key string) (err error) {
 	if err != nil {
 		return err
 	}
+	singleClusterIngresses, err := lbc.ingLister.ListGCEIngresses()
```

Comment:
```
gceIngresses, err := lbc.ingLister.LIstGCEIngress()
gceMutliIngresses, err := lbc.ingLister.ListGCEMultiClusterIngress()
combinedIngresses := append(gceIngresses, gceMutliIngresses)

gceIngressNodeports := lbc.tr.toNodePorts(gceIngresses)
combinedNodeports := lbc.tr.toNodePorts(combinedIngresses)
```

Reply:
renamed to gceIngresses, gceNodePorts and allIngresses, allNodePorts.